src/seedu/addressbook/data/person/AddressBlock.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+public class AddressBlock {
+    
+    private String block;
+    
+    /**
+     * Default constructor of AddressBlock
+     */
+    public AddressBlock() {}
+    
+    /**
+     * Constructor to initialise the block of the address
+     * @param block
+     */
+    public AddressBlock(String block) {
```

Comment:
How would you recommend to validate the block other than checking that it is not empty?